5calls/app/src/main/java/org/a5calls/android/a5calls/MainActivity.java

```diff
@@ -248,68 +244,105 @@ public void updateIssue(Issue issue) {
+                        holder.updateZipUi(true, 0);
+                    }
+                });
+            } else {
+                final Issue issue = mIssues.get(position - 1);
+
+                final IssueViewHolder holder = (IssueViewHolder) viewHolder;
+
+                holder.name.setText(issue.name);
+                holder.itemView.setOnClickListener(new View.OnClickListener() {
+                    @Override
+                    public void onClick(View v) {
+                        Intent issueIntent = new Intent(holder.itemView.getContext(),
+                                IssueActivity.class);
+                        issueIntent.putExtra(IssueActivity.KEY_ISSUE, issue);
+                        issueIntent.putExtra(IssueActivity.KEY_ZIP, mZip);
+                        startActivityForResult(issueIntent, ISSUE_DETAIL_REQUEST);
+                    }
+                });
+                int totalCalls = issue.contacts.length;
+                List<String> contacted = AppSingleton.getInstance(getApplicationContext())
+                        .getDatabaseHelper().getCallsForIssueAndZip(issue.id, mZip);
+                int callsLeft = totalCalls - contacted.size();
+                if (callsLeft == totalCalls) {
+                    if (totalCalls == 1) {
+                        holder.numCalls.setText(getResources().getString(R.string.call_count_one));
+                    } else {
+                        holder.numCalls.setText(String.format(
+                                getResources().getString(R.string.call_count), totalCalls));
+                    }
                 } else {
                     holder.numCalls.setText(String.format(
-                            getResources().getString(R.string.call_count), totalCalls));
+                            getResources().getString(R.string.call_count_remaining), callsLeft,
+                            totalCalls));
                 }
+                holder.doneIcon.setVisibility(callsLeft == 0 ? View.VISIBLE : View.GONE);
+            }
+        }
+
+        @Override
+        public int getItemViewType(int position) {
+            if (position == 0) {
+                return HEADER_TYPE;
             } else {
-                holder.numCalls.setText(String.format(
-                        getResources().getString(R.string.call_count_remaining), callsLeft,
-                        totalCalls));
+                return ISSUE_TYPE;
             }
-            holder.doneIcon.setVisibility(callsLeft == 0 ? View.VISIBLE : View.GONE);
         }
 
         @Override
-        public void onViewRecycled(IssueViewHolder holder) {
+        public RecyclerView.ViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
+            switch (viewType) {
+                case HEADER_TYPE:
+                    return new HeaderViewHolder(LayoutInflater.from(parent.getContext()).inflate(R.layout.header_issues_list_row, parent, false));
+                default:
```

Comment:
Instead of "default", I'd suggest explicitly checking ISSUE_TYPE, just in case we add more things to this later.